moveit_core/robot_model/src/joint_model_group.cpp

```diff
@@ -831,5 +831,41 @@ bool JointModelGroup::isValidVelocityMove(const double* from_joint_pose, const d
 
   return true;
 }
+
+std::pair<Eigen::VectorXd, Eigen::VectorXd> getLowerAndUpperLimits(const moveit::core::JointModelGroup& group)
+{
+  // Get the group joints lower/upper position limits.
+  Eigen::VectorXd lower_limits(group.getActiveVariableCount());
+  Eigen::VectorXd upper_limits(group.getActiveVariableCount());
+  const moveit::core::JointBoundsVector& group_bounds = group.getActiveJointModelsBounds();
+  int joint_index = 0;
```

Comment:
Would `variable_index` be more correct?

Reply:
I like joint index better

Reply:
The reason why I'm proposing `variable_index` is because a joint can have multiple variables and this function iterates over those variables, not over joints.

Reply:
I'll rename it